doc/nrf/app_dev/device_guides/nrf70/features.rst

```diff
@@ -55,3 +55,36 @@ The nRF70 Series devices also support the following functionalities:
 Peer-to-peer support in the form of Wi-Fi Direct® will be available in the future.
 
 See the :ref:`ug_wifi` documentation for more information related to Wi-Fi modes of operation.
+
+hostap
+******
+
+The nRF70 Series devices use the `WPA Supplicant`_ to implement full Wi-Fi functionality.
+The WPA supplicant is part of the ``hostap`` project and is a widely used implementation of the IEEE 802.11i standard for wireless LAN security.
+The WPA supplicant is a software component that implements the WPA, WPA2 and WPA3 security protocols.
+
+The nRF70 Series devices use `Zephyr hostap fork`_ a fork of the ``hostap`` project that is integrated with the Zephyr RTOS.
+The WPA supplicant is integrated with the Zephyr RTOS and registers as a Wi-Fi network manager in the Zephyr networking stack, see `Zephyr Wi-Fi NM API`_ for details.
+The `Zephyr Wi-Fi management`_ layer in Zephyr uses the Wi-Fi network manager to manage the Wi-Fi interface.
+
+The nRF70 Series drivers registers as a Wi-Fi device in the Zephyr networking stack and provides the Wi-Fi interface to the WPA supplicant.
```

Comment:
```suggestion
The nRF70 Series drivers register as a Wi-Fi device in the Zephyr networking stack and provide the Wi-Fi interface to the WPA supplicant.
```

Reply:
Modified from drivers->driver and didn't do the other changes